generator/src/test/java/com/scottlogic/deg/generator/cucumber/AValid.feature

```diff
@@ -146,46 +141,13 @@ Feature: User can specify that a field must be a valid ISIN (International Secur
     Then I am presented with an error message
     And no data is created
 
-
   @ignore
   Scenario: Running an 'aValid' request alongside a null constraint should fail with an error message
     Given foo is a valid "ISIN"
     And foo is null
     Then I am presented with an error message
     And no data is created
 
-
```

Comment:
Was this intentionally removed?

Reply:
Yes, this is a duplicate test found in the aValid.feature file and is redundant.